Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.12-stretch as builder
+MAINTAINER Matteo Scandolo <teo.punto@gmail.com>
+
+WORKDIR /go
+ENV GO111MODULE=on
+ADD . /go/src/github.com/fsouza/fake-gcs-server
```

Comment:
Since we're using go modules, this doesn't need to be added to the GOPATH, so we could do something like:

```dockerfile
WORKDIR /code
ADD . ./
RUN CGO_ENABLED=0 GOOS=linux go build -o /build/go-fake-storage
```

We can probably set GOPROXY to ``https://proxy.golang.org`` to speed up the build as well.